### DISCUSSION TITLE
Localisation feedback

### DISCUSSION THREAD

**Discussion by joshhunt** [grafana/grafana#59227] [General]
In Grafana 9.3 we have added 4 new languages to Grafana: Spanish, French, German and Simplified Chinese.

With millions of users across the globe, Grafana has a global footprint. In order to make it accessible to a wider audience, we have taken the first steps in localizing key workflows. You can now set Grafana’s language for the navigation, viewing dashboards, and a handful of settings.

It would be great to get feedback on what areas of Grafana are important for you for us to work on translating and localising next.

As for supported languages, Grafana will be handling the translations with our translation partners, so we are not yet accepting contributions for translations. This is something we would like to accept, through a platform such as Crowdin, in the future, but we have no immediate plans for it.

For any issues with existing translations, please raise an _Issue_ to this repository so we can triage and investigate appropriately.

**Comment by DmitrySidorow** [comment]
Russian language please

**Comment by kajla** [comment]
Hungarian language please.
We have a professional translation team.

**Comment by danmx** [comment]
Polish would be good too

**Comment by krzd7** [comment]
Can we have brazilian portuguese (pt-BR) added?

**Reply by TaylorHo** [reply]
I am ready if Grafana's Team need someone to carry out the translations. I can open a PR for that.

**Comment by 4lenz1** [comment]
Japanese language please

**Comment by profijoeln** [comment]
Finnish language support would be greatly appreciated!

**Comment by krzd7** [comment]
Please, localise user related emails such as user password reset as per #60027

**Comment by igiornotigi** [comment]
I hope the Italian language will also be included! 🍝 ❤

**Comment by krzd7** [comment]
Hello @joshhunt , 

Any ETA on supporting new languages?

**Comment by yosiasz** [comment]
is the plan to use something like the following (i18)?

https://locize.com/blog/react-i18next/

**Comment by Mateorid** [comment]
Localization for plugins please. There is currently no way to translate them dynamically.
And for dashboard variables.
Add a global variable for the selected language so it can be used in payloads.

**Comment by btbf** [comment]
japanese please

**Comment by jensb** [comment]
I find that l10n is actually more important than i18n. Date formats in legends, comma vs. dot for decimals, all these can be much more misleading than some vocabulary that users need to learn. "3/10" can be March 10, or October 3, or 0.3, or "0,3", depending on where you are.

I did not try 9.3 yet, but I would prioritize localized date and number formats for tables, labels and legends much higher than trying to i18n every string in the app. (Also, a better mobile UX, but that's a different topic.)

**Comment by ahmetcanterlikci** [comment]
Turkish would be great! I'd also like to help

**Comment by Exudius** [comment]
Hello,
We can help translate for Italian langauge, we really enjoy your platform and our team has been using it these past months.

**Comment by dg-hasimoto** [comment]
Japanese language please.
If you don't have a Japanese translation team, we can help translate for japanese language !

**Comment by parumsancto** [comment]
ukrainian please

**Comment by alerts-ua** [comment]
I would greatly appreciate support for the Ukrainian language!

**Comment by baur** [comment]
Is there a release roadmap with new languages? or future plans with internationalization?

**Comment by austy246** [comment]
Czech language would be highly appreciated. Thanks

**Comment by natellium** [comment]
Hello! We want to let you all know that we are moving to Github Issues for feature requests and forum posts. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

This discussion was converted into a forum post, which is available [here](https://community.grafana.com/t/localisation-feedback/101488)
